frontend/src/components/user-management/student/AddClassroomModal.tsx

```diff
@@ -151,13 +155,10 @@ const AddClassroomModal = ({
                   />
                 </VStack>
                 <VStack align="left" direction="column" width="320px">
-                  <FormLabel color="blue.300">School Year</FormLabel>
-                  <Input
-                    defaultValue="2023"
-                    onChange={(e) => handleChange(e, "schoolYear")}
-                    placeholder="Type in School Year"
-                    type="number"
-                    value={watch("schoolYear")}
+                  <FormLabel color="blue.300">Start Date</FormLabel>
+                  <DatePicker
+                    onChange={handleDateChange}
+                    value={watch("startDate")}
```

Comment:
do we check somewhere that the `startDate` is non-empty and on or after the current date?